.github/workflows/build-manywheel-images.yml

```diff
@@ -37,15 +37,15 @@ jobs:
     runs-on: linux.12xlarge.ephemeral
     strategy:
       matrix:
-        cuda_version: ["12.1", "11.8"]
+        cuda_version: ["11.8", "12.1", "12.4"]
```

Comment:
Please see https://github.com/pytorch/builder/pull/1726

Reply:
thank you! reverted this change in https://github.com/pytorch/builder/pull/1734/commits/e5c34cadf79b956f5116a7a5e107f6c9ab207df2